tests/AsyncBridge.Tests/ContinueWithTests.cs

```diff
@@ -0,0 +1,314 @@
+        {
+            var GotState = false;
+            TestUtils.RunAsync(() => TaskEx.Run(() => { }).ContinueWith((task, state) => GotState = (bool)state, true));
+            Assert.IsTrue(GotState);
+        }
+
+        [TestMethod, ExpectedException(typeof(ArgumentNullException))]
+        public void ActionNull()
+        {
+            TaskEx.Run(() => { }).ContinueWith((Action<Task, object>)null, false);
+        }
+
+        [TestMethod]
+        public void ActionToken()
+        {
+            var GotState = false;
+            TestUtils.RunAsync(() => TaskEx.Run(() => { }).ContinueWith((task, state) => GotState = (bool)state, true, CancellationToken.None));
+            Assert.IsTrue(GotState);
+        }
+
+        [TestMethod, ExpectedException(typeof(ArgumentNullException))]
+        public void ActionTokenNull()
+        {
+            TaskEx.Run(() => { }).ContinueWith((Action<Task, object>)null, false, CancellationToken.None);
+        }
+
+        [TestMethod]
+        public void ActionOptions()
+        {
+            var GotState = false;
+            TestUtils.RunAsync(() => TaskEx.Run(() => { }).ContinueWith((task, state) => GotState = (bool)state, true, TaskContinuationOptions.None));
+            Assert.IsTrue(GotState);
+        }
+
+        [TestMethod, ExpectedException(typeof(ArgumentNullException))]
+        public void ActionOptionsNull()
+        {
+            TaskEx.Run(() => { }).ContinueWith((Action<Task, object>)null, false, TaskContinuationOptions.None);
+        }
+
+        [TestMethod]
+        public void ActionScheduler()
+        {
+            var GotState = false;
+            TestUtils.RunAsync(() => TaskEx.Run(() => { }).ContinueWith((task, state) => GotState = (bool)state, true, TaskScheduler.Current));
+            Assert.IsTrue(GotState);
+        }
+
+        [TestMethod, ExpectedException(typeof(ArgumentNullException))]
+        public void ActionSchedulerNull()
+        {
+            TaskEx.Run(() => { }).ContinueWith((Action<Task, object>)null, false, TaskScheduler.Current);
+        }
+
+        [TestMethod]
+        public void ActionTokenOptionsScheduler()
+        {
+            var GotState = false;
+            TestUtils.RunAsync(() => TaskEx.Run(() => { }).ContinueWith((task, state) => GotState = (bool)state, true, CancellationToken.None, TaskContinuationOptions.None, TaskScheduler.Current));
+            Assert.IsTrue(GotState);
+        }
+
+        [TestMethod, ExpectedException(typeof(ArgumentNullException))]
+        public void ActionTokenOptionsSchedulerNull()
+        {
+            TaskEx.Run(() => { }).ContinueWith((Action<Task, object>)null, false, CancellationToken.None, TaskContinuationOptions.None, TaskScheduler.Current);
+        }
+
+        [TestMethod]
+        public void ActionResult()
+        {
+            var GotState = false;
+            TestUtils.RunAsync(() => TaskEx.Run(() => true).ContinueWith((task, state) => GotState = (bool)state, true));
```

Comment:
Same deal, testing the `Func<Task<bool>, object, bool>` overload instead of the `Action<Task<bool>, object>` overload.

https://codecov.io/gh/OmerMor/AsyncBridge/src/2806bdff8e65cfb4be247c1595fc319cdc3baa19/src/AsyncBridge/AsyncCompatLibExtensions.cs#L799...817

(5 places)